assets/src/components/detoursTable.tsx

```diff
@@ -16,27 +16,29 @@ export const DetoursTable = ({ data }: DetoursTableProps) => {
     <Table hover className="c-detours-table">
       <thead>
         <tr>
-          <th>Route</th>
-          <th className="u-hide-for-mobile">Starting Intersection</th>
-          <th className="u-hide-for-mobile">On detour since</th>
+          <th className="px-3 py-4">Route and direction</th>
```

Comment:
I'd be interested to pick apart my instincts/choices here. My general strategy was to do everything that I could easily do using bootstrap utilities, and then put the rest into CSS.

I don't LOVE this because it means that our visual/styling logic is kind of spread out between CSS and HTML, but it does reduce CSS bundle size and lines up more closely with how we've been doing things so far.

Plus, when they work, the bootstrap utilities are quite nice!